chain/sync.go

```diff
@@ -980,36 +980,35 @@ func (syncer *Syncer) collectHeaders(ctx context.Context, from *types.TipSet, to
 		}
 	}
 
-	{
-		// ensure consistency of beacon entires
-		targetBE := from.Blocks()[0].BeaconEntries
-		if len(targetBE) == 0 {
-			syncer.bad.Add(from.Cids()[0], "no beacon entires")
-			return nil, xerrors.Errorf("block (%s) contained no drand entires", from.Cids()[0])
-		}
-		cur := targetBE[0].Round
-
-		for _, e := range targetBE[1:] {
-			if cur >= e.Round {
-				syncer.bad.Add(from.Cids()[0], "wrong order of beacon entires")
-				return nil, xerrors.Errorf("wrong order of beacon entires")
-			}
+	/*
+		{
+			// TODO: Not sure what this check is aiming to do exactly, but it doesnt quite work
```

Comment:
This is ensuring all blocks in tipset have the same beacon entries in them.

Reply:
Ah, but blocks can not have drand entries in them some of the time.

Also this looks like its only checking a single tipset for consistency here, am i missing something?